### ISSUE TITLE
Add more comments around the code to make it friendlier

### ISSUE THREAD

**Issue by Vizualni** [Vizualni/whoops#2]
As the title says :)

**Comment by wilrodriguez** [comment]
Yes, please! I just found this library and it's making my life a lot easier but my team wants to see it more documented before they decide to jump in on it as well.
